### PR TITLE
Add simple local FTP server for development of ftp-watcher

### PR DESCRIPTION
**Pull request by theefer**
Had this locally for a while, can be handy for others.

### CODE REVIEW

File: dev-ftp/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Development FTP server
+
+Simple local FTP server to help test the `ftp-watcher` service in
+development.
+
+To use this development FTP server, add the following lines to your
+`/etc/gu/ftp-watcher.properties` file:
```
Comment (akash1810) [inline dev-ftp/README.md]: worth adding to the generator?

File: dev-ftp/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ftp.host=localhost
+ftp.port=41756
+ftp.user=any
+ftp.password=any
+```
+
+Install the necessary dependencies:
+
+```
+$ gem install ftpd
```
Comment (akash1810) [inline dev-ftp/README.md]: Considered using `bundler` to allow [`bundle install`](http://bundler.io/v1.7/rationale.html#checking-your-code-into-version-control)?

Possibly not worth it as its one dependancy...

Comment (theefer) [inline dev-ftp/README.md]: Yeah I thought it's a bit overkill, can add if people disagree

/cc @kenoir who knows about Ruby things

Comment (kenoir) [inline dev-ftp/README.md]: It's not. You'd have to install `bundler` first.

**Comment by theefer** [comment]
@akash1810 ping now using dot-properties!

File: dev-ftp/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Development FTP server
+
+Simple local FTP server to help test the `ftp-watcher` service in
+development.
+
+To use this development FTP server, register an API key for your local
+server and add it to your `/etc/gu/ftp-watcher.properties` file (you
+can use the dot-properties helper script and configure its `ftp.key`).
```
Comment (akash1810) [inline dev-ftp/README.md]: `ftp_key`

**Comment by akash1810** [comment]
:+1:

File: dev-ftp/README.md (hunk 3 above)
Comment (theefer) [inline dev-ftp/README.md]: Fixed ta!

**Pull request merged**